src/components/profile/ProfileDescription.vue

```diff
@@ -9,6 +9,8 @@
 import marked from 'marked';
 import insane from 'insane';
 
+insane.defaults.allowedTags = ['a', 'em', 'li', 'ol', 'p', 'strong', 'ul'];
```

Comment:
Do you know how this translates into markdown?

Reply:
As in, what markdown we're supporting?

It's just:

`[a](http://example.com)`
`*em*`
```
* <ul>
* <li>
```
```
1. <ol>
2. <li>
```
```
p

p
```
`**strong**`

Reply:
maybe add `pre` and `code` for \`\`\` and \`